test/e2e/login.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+
+test('Login with User 1', async ({ page }) => {
+  await page.goto('/login?redirectTo=%2F');
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Username').fill('user1');
+  await page.getByLabel('Password').fill('user1pass');
+  await page.getByRole('button', { name: 'Login' }).click();
+  await page.waitForURL('http://localhost:8080');
+  const locator = page.getByText('user1', { exact: true });
+  const starchartHeading = page.getByRole('heading', { name: 'Starchart' });
+  await expect(locator).toContainText('user1');
+  await expect(starchartHeading).toContainText('Starchart');
+});
+
+test('Login with User 1 dev redirect', async ({ page }) => {
+  await page.goto('/login?redirectTo=%2Fdev');
```

Comment:
We are likely to remove `/dev` at some point, so I'd use `/domains` or something that is more stable.